### ISSUE TITLE
Linking production copies of libraries

### ISSUE THREAD

**Issue by chadbrewbaker** [awslabs/aws-lambda-cpp#22]
LambCi, just vacuuming the binaries from a live AWS Lambda instance, is a bit of a hack.

Is there any repository to pull AWS Lambda instance dynamic/static libraries from directly to remove the Docker repo dependency on builds? Bring your own libc isn't enough. Openssl etc have had breaking changes too.

**Comment by marcomagdy** [comment]
I'm not sure I understand the problem. There is no Docker repo dependency.
OpenSSL binaries that your lambda links too are packaged automatically as well as libc.

Can you rephrase the issue you're referring to?
